session_3/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Articles</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Tags</th>
+      <th>Topic</th>
+      <th>Content</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @articles.each do |article| %>
+      <tr>
+        <td><%= article.title %></td>
+        <td><%= article.tags %></td>
+        <td><%= article.topic %></td>
+        <td><%= article.content %></td>
+        <td><%= link_to 'Show', article %></td>
```

Comment:
Enclose the links with `if can?` tags, otherwise anyone who can view the article might see buttons but the buttons won't do anything